src/Illuminate/Contracts/Database/Eloquent/Castable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Illuminate\Contracts\Database\Eloquent;
+
+interface Castable
```

Comment:
The docblocks in this interface suggests that we are working with a model somehow ("from the model" and "on the model"), but no model is given in any of the declarations.

Reply:
Castes are used only when processing model values.
An interface is needed both to determine the required methods and to verify the correctness of their instructions.
See https://github.com/andrey-helldar/framework/blob/patch-casts-manage/src/Illuminate/Database/Eloquent/Concerns/HasCasts.php#L273-L280